spec/20-http_header_format.md

```diff
@@ -280,8 +280,7 @@ lcalpha    = %x61-7A ; a-z
 
 **Note**: Identifiers MUST begin with a lowercase letter or a digit, and can only contain lowercase letters (`a`-`z`), digits (`0`-`9`), underscores (`_`), dashes (`-`), asterisks (`*`), and forward slashes (`/`).
 
-For multi-tenant vendor scenarios, an at sign (`@`) can be used to prefix the vendor name. Vendors SHOULD set the tenant ID at the beginning of the key. For example, \
-`fw529a3039@dt` - `fw529a3039` is a tenant ID and `@dt` is a vendor name. Searching for `@dt=` is more robust for parsing (for example, searching for all a vendor's keys).
+In a distributed application where different components may be traced by different instances of the same multi-tenant tracing system, the system SHOULD keep the system ID at the end of the key. For example, `001@xyz` where `001` is the ID of the specific tenant within the tracing system and `xyz` is the tracing system identifier. This makes for fast and robust parsing. For example, by searching for all instances of `@xyz=` a tracing system can easily find all keys made by system `xyz`.
```

Comment:
Should we describe more about the @ character: is that an example or are we specifically saying that tracing systems SHOULD use @ as the separator?

Reply:
+1 to this. We only allow one `@` sign for this reason

Reply:
since it's errata, may not ned to change the ABNF definition. But in main we can even call section after the `@` sign as `tracing-system` explicitly

Reply:
In main we currently describe _less_ about the `@` character and we allow more than one. For this change, I can describe more what its meaning is, but it was removed for a reason I assume.

Reply:
>  we allow more than one. 

do we? We only allow one per key, correct?

Reply:
No, we don't have any restrictions in `main`